report-jtreg-comparator/src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/arguments/ArgumentsParsing.java

```diff
@@ -199,7 +209,7 @@ private static Options.Configuration getConfiguration(String[] values, String cu
     }
 
     private static String getArgumentValue(String[] arguments, int i) {
-        if (i + 1 < arguments.length) {
+        if (i + 1 < arguments.length && !arguments[i + 1].matches("^--.*")) {
             return arguments[i + 1];
         } else {
             throw new RuntimeException("Expected some value after " + arguments[i] + " argument.");
```

Comment:
trust in you, I have, young padawan.  However this is becaoming regex hell :) I once implemented mathematic formula parser based on split and repalce... not any more!